app/assets/javascripts/votes.js

```diff
@@ -0,0 +1,50 @@
+$( "a.upvote i" ).click(function( event ) {
+  event.preventDefault();
+  event.stopPropagation();
+  $(this).toggleClass("gold");
+  var reviewId = $(this).attr("id").match(/review-([0-9]+)/)[1];
```

Comment:
You could create a dynamic id in your view based on each review and grab those elements. I wouldn't change this much right now, but might be worth adding an issue your github repo to refactor later on.